Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -20,23 +20,34 @@ public class BoundaryManager : Singleton<BoundaryManager>
         [Tooltip("Quad prefab to display as the floor.")]
         public GameObject FloorQuad;
         private GameObject floorQuadInstance;
-        public float CurrentFloorHeightOffset { get { return newFloorHeight.y; } }
-        private Vector3 newFloorHeight;
 
 #if UNITY_WSA && UNITY_2017_2_OR_NEWER
         [SerializeField]
         [Tooltip("Approximate max Y height of your space.")]
         private float boundaryHeight = 10f;
+
         private Bounds boundaryBounds;
+
+        [SerializeField]
+        // Defaulting coordinate system to RoomScale in immersive headsets.
+        // This puts the origin (0, 0, 0) on the floor if a floor has been established during setup via MixedRealityPortal.
+        private TrackingSpaceType opaqueTrackingSpaceType = TrackingSpaceType.RoomScale;
```

Comment:
These made the code hard to read but okay.

Reply:
This allows the dev to select which tracking space type they want, to more easily support seated-experience apps like MixedReality250.

Reply:
MR250 still sets it as full scale  for immersive.